include/oneapi/dpl/internal/dynamic_selection_impl/dynamic_load_policy.h

```diff
@@ -14,6 +14,8 @@
 #include <memory>
 #include <ostream>
 #include <vector>
+#include <exception>
```

Comment:
ostream should be removed. And I think mutex and limits are missing.

Reply:
Done